packages/entities/entities-plugins/sandbox/pages/PluginSelectPage.vue

```diff
@@ -3,6 +3,7 @@
     <h2>Konnect API</h2>
     <PluginSelect
       :config="konnectConfig"
+      :disabled-plugins="{ 'acl': 'ACL is not supported for this entity type'}"
```

Comment:
question: seems unrelated?

Reply:
Using it to make sure I didn't break display of disabled plugins in the sandbox